dynamics_class: "quad_near_hover"

safety_states: 
  - 1
  - 2
  - 4
  - 5
safety_controls:
  - 0
  - 3 

obstacles: 
  
  obstacle1:
    type: 'Rectangle'
    mode: 'Active'
    indices:
      - 0
      - 1
    minVal:
      - -1.0
      - 1.0
    maxVal:
      - 2.0
      - 3.0
    padding: .20


boundary:
  padding: 0
  indices:
    - 0
    - 1
  minVal:
    - -1.0
    - 0.0
  maxVal:
    - 8.0
    - 5.0

obstacle_update_rate_hz: 5

state_domain:
  lo:
    - -6.0
    - -0.0
    - -5.0
    - -5.0
  hi:
    - 6.0
    - 4.1
    - 5.0
    - 5.0
  resolution:
    - 31
    - 31
    - 25
    - 25
  periodic_dims: []
  
control_space:
  n_dims: 2
  lo:
    - -0.1
    - 0.0
  hi: 
    - 0.1
    - 12.0

disturbance_space:
  n_dims: 
  lo: 0
  hi: 0

actuation_updates: []

disturbance_updates: []